sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -600,6 +602,13 @@ abstract class Element<T extends Widget> implements BuildContext {
 typedef Widget WidgetBuilder(BuildContext context);
 typedef void BuildScheduler(BuildableElement element);
 
+class ErrorWidget extends LeafRenderObjectWidget {
+  RenderBox createRenderObject() => new RenderErrorBox();
+
+  void updateRenderObject(RenderObject renderObject, RenderObjectWidget oldWidget) {
+  }
```

Comment:
this class is so trivial, i'd remove the blank line and put the `{ }` on one line.